hc/api/helpers.py

```diff
@@ -0,0 +1,7 @@
+from django.db.models import Q
+
+
+def get_running_checks(query):
```

Comment:
I'm not sold into this whole concept of using helpers that are not tied to any classes. Could you move this to a proper class? We can chat more at length about this person if this is not clear